README.md

```diff
@@ -269,3 +269,21 @@ export const lookId = 1
 * The server will print out what host and port it is running on.
 
 You may need to `pip install six` to install the Python 2/3 compatibility layer.
+
+### Troubleshooting
```

Comment:
@whscullin Questions for updating docs:

1. Should Logging (currently H2) be a subheading of Troubleshooting (currently H3)?

2. In Docs we have a separate page for [SDK Intro](https://docs.looker.com/reference/embed-sdk/embed-sdk-intro) and [SDK demo](https://docs.looker.com/reference/embed-sdk/embed-sdk-demo) I'm going to chuck your new Logging section on the intro page, would you agree that it's a generic section for SDK and not specific to the demo?

Reply:
1. Markdown - how does it work?

2. Definitely generic.

Reply:
Also, I got busted by our editor while copying your work:

![image](https://user-images.githubusercontent.com/25802439/90065234-d08cfc00-dca0-11ea-95a4-34d34612cb26.png)


Would that be a command?  Is that update correct?

Reply:
I guess that works. Programmatically speaking it's a statement, I think, but it's probably more clearly conveyed as a command. Perhaps why I just avoided taxonomy.

Reply:
I was also trying to avoid, but I got called out. I guess some people think it's my "job" to "be clear when writing."

Reply:
The temerity of some people

Reply:
The gall.